pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -650,7 +650,7 @@ func logMigrationInfo(logger *log.FilteredLogger, uid string, info *libvirt.Doma
 	}
 
 	bToMbps := func(bytes uint64) uint64 {
```

Comment:
The units on both sides of the conversion are not the same.
In case this function converts bytes per second to mega bits per second, please consider renaming it.